src/videojs-http-streaming.js

```diff
@@ -377,6 +377,20 @@ class HlsHandler extends Component {
       }
     });
 
+    const getTimeRanges = (player, type) => {
```

Comment:
Since player and type are only used once to get the time ranges, it might be simpler and clearer to only pass in the time ranges themselves. The function may also be better served as being called `timeRangesToArray` and if we put it outside of the function we can add some tests for it.